source/renderer/app/components/widgets/forms/InlineEditingInput.js

```diff
@@ -1,14 +1,23 @@
 // @flow
+/* eslint-disable react/no-did-update-set-state */
 import React, { Component } from 'react';
 import { observer } from 'mobx-react';
 import { defineMessages, intlShape } from 'react-intl';
+import { get } from 'lodash';
+import { Button } from 'react-polymorph/lib/components/Button';
+import { ButtonSkin } from 'react-polymorph/lib/skins/simple/ButtonSkin';
 import vjf from 'mobx-react-form/lib/validators/VJF';
+import SVGInline from 'react-svg-inline';
 import classnames from 'classnames';
 import { Input } from 'react-polymorph/lib/components/Input';
 import { InputSkin } from 'react-polymorph/lib/skins/simple/InputSkin';
```

Comment:
@daniloprates You don't need to import and apply InputSkin.
It is used by default, so please remove this.